starter-core/src/main/java/io/micronaut/starter/feature/build/maven/templates/aotDependencies.rocker.raw

```diff
@@ -6,7 +6,6 @@
                   <dependency>
                       <groupId>@(aotDependency.getGroupId())</groupId>
                       <artifactId>@(aotDependency.getArtifactId())</artifactId>
-                      <version>@(aotDependency.getVersion())</version>
```

Comment:
This should be `<version>${micronaut.security.version}</version>`

Reply:
Ahhhh so then it comes from platform parent which is similar to what we're doing with Gradle